src/scancode/agent/scancode_wrapper.hpp

```diff
@@ -0,0 +1,41 @@
+/*****************************************************************************
+ * SPDX-License-Identifier: GPL-2.0
+ * SPDX-FileCopyrightText: 2021 Sarita Singh <saritasingh.0425@gmail.com>
+ * 
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * version 2 as published by the Free Software Foundation.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ ****************************************************************************/
+
+#ifndef SCANCODE_AGENT_SCANCODE_WRAPPER_HPP
+#define SCANCODE_AGENT_SCANCODE_WRAPPER_HPP
+
+#define AGENT_NAME "scancode"
+#define AGENT_DESC "scancode agent"
+#define AGENT_ARS  "scancode_ars"
+
+#include <string>
+#include <vector>
+#include<unordered_map>
+#include "files.hpp"
+#include "match.hpp"
+#include "scancode_state.hpp"
+#include <map>
+#include <jsoncpp/json/json.h>
```

Comment:
Please order include statements. Standard libs should come first, then third party, then from project. There should be 1 empty line between each group.